### PR TITLE
Implement commits page

### PR DESCRIPTION
**Pull request by abrl91**
Issue Number: #3447

## PR Details

commit page - list of commits and their builds

## PR Checklist
- [ ] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

### CODE REVIEW

File: packages/amplication-client/src/Resource/code-view/CodeViewExplorer.tsx [hunk 1]
```diff
@@ -34,7 +34,7 @@ const CodeViewExplorer: React.FC<Props> = ({ resource, onFileSelected }) => {
     setSelectedBuild(build);
     onFileSelected(null);
   };
-
+
```
Comment (barshimi) [inline packages/amplication-client/src/Resource/code-view/CodeViewExplorer.tsx:37]: remove space

File: packages/amplication-client/src/VersionControl/Commit.tsx [hunk 2]
```diff
@@ -62,7 +61,7 @@ const Commit = ({ projectId, resourceId, noChanges }: Props) => {
       {
         query: GET_BUILDS_COMMIT,
         variables: {
-          resourceId,
+          resourceId: "",
```
Comment (barshimi) [inline packages/amplication-client/src/VersionControl/Commit.tsx:64]: why ?

File: packages/amplication-client/src/VersionControl/CommitListItem.scss [hunk 3]
```diff
@@ -1,8 +1,28 @@
...
+  &__data {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__metadata {
+    display: flex;
+    flex-direction: column;
+    width: 130px;
```
Comment (barshimi) [inline packages/amplication-client/src/VersionControl/CommitListItem.scss:13]: params or variables

File: packages/amplication-client/src/VersionControl/CommitListItem.scss [hunk 4]
```diff
@@ -1,8 +1,28 @@
...
+    padding: 0 10px;
```
Comment (barshimi) [inline packages/amplication-client/src/VersionControl/CommitListItem.scss:14]: also here

File: packages/amplication-client/src/VersionControl/CommitListItem.tsx [hunk 5]
```diff
@@ -1,76 +1,38 @@
...
-  const handleRowClick = useCallback(() => {
-    history.push(
-      `/${currentWorkspace?.id}/${currentProject?.id}/${resourceId}/commits/${commit.id}`
-    );
-  }, [history, resourceId, commit, currentWorkspace, currentProject]);
-
-  const account = commit.user?.account;
+  const { currentWorkspace } = useContext(AppContext);
 
+  // const account = commit.user?.account;
```
Comment (barshimi) [inline packages/amplication-client/src/VersionControl/CommitListItem.tsx:20]: remove comment

File: packages/amplication-client/src/VersionControl/CommitResourceListItem.scss [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+    &:last-child {
+      margin-bottom: 0;
+    }
+
+    .clickable-id {
+      color: var(--black80);
+      a {
+        color: var(--black100);
+        margin-left: 8px;
```
Comment (barshimi) [inline packages/amplication-client/src/VersionControl/CommitResourceListItem.scss:24]: variable

File: packages/amplication-client/src/VersionControl/CommitResourceListItem.scss [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+    }
+  }
+
+  &__title {
+    margin-right: auto;
+    display: flex;
+    align-items: center;
+
+    p {
+      padding: 0 8px;
```
Comment (barshimi) [inline packages/amplication-client/src/VersionControl/CommitResourceListItem.scss:39]: variable

File: packages/amplication-client/src/VersionControl/LastCommit.tsx [hunk 8]
```diff
@@ -131,6 +131,13 @@ function formatTimeToNow(time: Date | null): string | null {
   );
 }
 
+// formatTimeToNow returns "about x time ago and I want to remove the word /about/"
```
Comment (barshimi) [inline packages/amplication-client/src/VersionControl/LastCommit.tsx:134]: add useCallback

File: packages/amplication-client/src/VersionControl/Commit.tsx (hunk 2 above)
Comment (abrl91) [reply packages/amplication-client/src/VersionControl/Commit.tsx:64]: hum, we forgot about it. Can we remove it?

**Review by barshimi — APPROVED** [review]

**Pull request merged**